charts/base-cluster/templates/monitoring/kube-prometheus-stack/_prometheus_config.yaml

```diff
@@ -43,11 +43,17 @@ prometheusSpec:
   probeSelector:
     matchLabels: {{- .Values.monitoring.labels | toYaml | nindent 6 }}
   additionalAlertRelabelConfigs:
+    {{- if .Values.monitoring.monitorAllNamespaces }}
+      {{- $namespaces := list .Release.Namespace "kube-node-lease" "kube-public" "kube-system" -}}
```

Comment:
So you can just monitor all or none?  
Or did i miss something?

Reply:
I think a seperation for the three namespaces manually added to the list and all other created ones would be good. Or maybe the possibility to simply pass a list,  overwriting the flag.

Reply:
Whoopsie, forgot the `not`